xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -443,20 +443,22 @@ void CGUIFontTTF::DrawTextInternal(CGraphicContext& context,
       // by taking in account that we have also to add ellipses before the text
       float textWidth = ellipsesWidth;
       // We need to iterate from the end to the beginning
-      auto itRGliph = glyphs.rbegin();
-      for (; itRGliph != glyphs.rend(); itRGliph++)
+      for (auto itRGliph = glyphs.rbegin(); itRGliph != glyphs.rend(); itRGliph++)
```

Comment:
isn't ++itRGliph preferred?

Reply:
hmm depends on the compiler, if compiler optimizations are disabled using prefix on "for each" is more fast, otherwise there is same performances with both prefix/postfix
however i will change with prefix just to be sure

Reply:
crbegin / crend? no difference on Windows but maybe for other platforms